resources/views/user/components/changelog_list.blade.php

```diff
@@ -14,23 +14,21 @@
                     @lang('importiert von')
                 @else
                     @unless(empty($changelog->changelog->get('changes', [])))
-                        <span
-                            @php
-                                $str = [];
-                                foreach($changelog->changelog['changes'] as $key => $change) {
-                                    if (is_array($change['old'])) {
-                                        $str[] = ucfirst($key).": ".implode(', ', $change['old'])." &rarr; ".implode(', ', $change['new']);
-                                    } else {
-                                        $str[] = ucfirst($key).": ".\Illuminate\Support\Str::limit($change['old'], 40, "&hellip;")." &rarr; ".\Illuminate\Support\Str::limit($change['new'], 40, "&hellip;");
```

Comment:
`&hellip;` or … marks that a part of a text is truncated. This should be used here